models/login_source.go

```diff
@@ -217,6 +247,8 @@ func (source *LoginSource) UseTLS() bool {
 		return source.LDAP().SecurityProtocol != ldap.SecurityProtocolUnencrypted
 	case LoginSMTP:
 		return source.SMTP().TLS
+	case LoginOAuth2:
+		return true
```

Comment:
Not sure how to use this property correct if we are doing a call to the OAuth2 provider, suggestions?